.hooks/README.md

```diff
@@ -0,0 +1,31 @@
+# Git Hooks
+
+From [Git docs](https://git-scm.com/docs/githooks#_description):
+
+> Hooks are programs you can place in a hooks directory to trigger actions at certain points in git’s execution. Hooks that don’t have the executable bit set are ignored.
+
+## Usage
+
+To use this hook, it needs to be linked into the `.git/hooks` directory and made execuable:
+
+```sh
+ln -s -f ../../.hooks/pre-commit ./.git/hooks/pre-commit
```

Comment:
where do I need to execute this command (what does `../..` refer to?)

So I get it right that the hook is not automatically active? Should we make it automatically active?

Reply:
It is not possible to activate it automatically as this is a client-side hook and resides in the `.git` folder which is not under version control.

The `../../` refers to the `.git/hooks/` folder where the link is placed.

Reply:
You execute it in the projects roof folder.